store/tikv/2pc.go

```diff
@@ -435,6 +439,66 @@ func (c *twoPhaseCommitter) doActionOnMutations(bo *Backoffer, action twoPhaseCo
 		return errors.Trace(err)
 	}
 
+	// Pre-split regions to avoid too much write workload into a single region.
+	// In the large transaction case, this operation is important to avoid TiKV 'server is busy' error.
+	var preSplited bool
+	for _, group := range groups {
+		if group.mutations.len() > preSplitThreshold || testSplitRegionFlag {
```

Comment:
I think it's better to make preSplitThreshold a variable and modify it in the test.